### ISSUE TITLE
Fails to configure for ParaView v5.9.0 with mpi

### ISSUE THREAD

**Issue by jspanchu** [topology-tool-kit/ttk#605]
**Describe the bug**
CMake 3.18.4 fails to even configure TTK(dev) with ParaView v5.9.0. I'm not sure if this is a bug in ParaView's cmake files or TTK's cmake. Afaik, it is not a TTK bug. @CharlesGueunet.  [Here's](https://gitlab.kitware.com/paraview/paraview/-/issues/20324) a similar error-report from ParaView's issues

**To Reproduce**
1. Build and install ParaView v5.9.0 with OpenMPI
2. cmake .
2. see error

**Screenshots**
note the warning and error
![image](https://user-images.githubusercontent.com/40538987/115519119-2e8ace00-a289-11eb-8bc0-94aac9b14c8a.png)

**System (please complete the following information):**
 - OS: CentOS 8.2.2004
 - Compiler GCC-10.2.0

**Additional context**
This seems to happen only when ParaView was built with MPI.

**Comment by pierre-guillou** [comment]
Hi @jaswantp, 

I've quickly investigated this issue and indeed, I got the same error.

A quick workaround I experimented is to delete the FindMPI.cmake file embedded in the ParaView installation (in your case, `/work/imag/opt/.../vtkm/cmake/FindMPI.cmake`). Doing so, CMake should rely on its own FindMPI.cmake, which differs from the one embedded in ParaView, and the error should go away (at least it did on my computer).

Pierre
